shared/bash_remediation_functions/include_lineinfile.sh

```diff
@@ -116,7 +116,7 @@ function only_lineinfile() {
     local insert_before="$6"
     local insensitive="$7"
 
-    lineinfile "$path" "$line_regex" '' 'false' 'absent' '' '' "$insensitive"
+    lineinfile "$path" "$line_regex" '' "$create" 'absent' '' '' "$insensitive"
```

Comment:
Isn't it strange to pass `create=true` and `state=absent`? It will create an empty file just to check that something is not there.
If the state is `absent` it makes sense that the file is not created.
What if instead of passing `$create` with `absent`, the function `lineinfile` in line 100, also checked `if state == "present"`?